src/app/layout.tsx

```diff
@@ -27,6 +27,50 @@ export default async function RootLayout({
         <meta name="twitter:description" content="sBTC Bridge" />
         <meta name="twitter:image" content="/images/icon.png" />
         <meta name="twitter:image:alt" content="sBTC Bridge" />
+        <script
+          dangerouslySetInnerHTML={{
+            __html: `
+              window.__lc = window.__lc || {};
+              window.__lc.license = 18945692;
+              window.__lc.integration_name = "manual_onboarding";
+              window.__lc.product_name = "livechat";
+              ;(function(n,t,c){
+                function i(n){return e._h?e._h.apply(null,n):e._q.push(n)}
+                var e={_q:[],_h:null,_v:"2.0",
+                  on:function(){i(["on",[].slice.call(arguments)])},
+                  once:function(){i(["once",[].slice.call(arguments)])},
+                  off:function(){i(["off",[].slice.call(arguments)])},
+                  get:function(){
+                    if(!e._h)throw new Error("[LiveChatWidget] You can't use getters before load.");
+                    return i(["get",[].slice.call(arguments)])
+                  },
+                  call:function(){i(["call",[].slice.call(arguments)])},
+                  init:function(){
+                    var n=t.createElement("script");
+                    n.async=!0,n.type="text/javascript",
+                    n.src="https://cdn.livechatinc.com/tracking.js",
```

Comment:
@matteojug using your JS-fu: this bit was blocked by my ad-blocker, so I did not see the support icon. That's unfortunate. But it seems like this is required for the chat thing to work, it's not just for "tracking" as the name suggests. Is this correct?